src/System.Private.CoreLib/shared/System/Text/Unicode/Utf8Utility.cs

```diff
@@ -24,6 +35,11 @@ internal static partial class Utf8Utility
         /// </summary>
         private static ReadOnlySpan<byte> ReplacementCharSequence => new byte[] { 0xEF, 0xBF, 0xBD };
 
+        /// <summary>
+        /// A <see cref="Vector128{SByte}"/> where each element has a value corresponding to its index.
+        /// </summary>
+        private static readonly Vector128<sbyte> VectorOfElementIndices = Vector128.Create(0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15);
```

Comment:
```suggestion
        private static readonly Vector128<sbyte> s_vectorOfElementIndices = Vector128.Create(0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15);
```
As declared, it's a field?